src/lib/log.js

```diff
@@ -2,6 +2,7 @@
  * lib/log.js
  * Creates a simple, multi environment, namespaced log class
  * NOTE: our global namespace is "houston"
+ * @flow
```

Comment:
Yes that enables flow checking on this file. One of the reasons I went with flow is it allows you to gradually add it to a project. I could enable it globally, but it would start complaining about the files that have yet to be converted.